pyccel/codegen/printing/pycode.py

```diff
@@ -845,6 +845,13 @@ def _print_NumpyCountNonZero(self, expr):
 
         return "{}({})".format(name, arg)
 
+    def _print_ListAppend(self, expr):
+        method_name = expr.name
+        dotted_var = self._print(expr.args[0])
+        method_args = self._print(expr.args[1])
```

Comment:
It would be nice to give `expr.args[0]` and `expr.args[1]` more explicit names. This can be done using the `@property` decorator with class methods